Dockerfile

```diff
@@ -45,12 +46,47 @@ RUN git clone https://github.com/crosstool-ng/crosstool-ng \
  && echo 'CT_CC_GCC_CORE_EXTRA_CONFIG_ARRAY="--enable-standard-branch-protection"' >>.config \
  && ./ct-ng build.$(nproc)
 
+RUN wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.16.8/zephyr-sdk-0.16.8_linux-x86_64.tar.xz
+
+RUN tar xvf zephyr-sdk-0.16.8_linux-x86_64.tar.xz
```

Comment:
Each RUN creates a layer, are you sure you want separate layers for download, unpack, and cleanup Zephyr SDK? I think it is better to combine